modules/__tests__/IndexLink-test.js

```diff
@@ -54,25 +85,57 @@ describe('An <IndexLink>', function () {
     React.unmountComponentAtNode(node)
   })
 
-  describe('when linking to the root', function () {
-    it("is active when the parent's route is active", function (done) {
+  describe('when linking to the overview', function () {
+    it("is active and other routes are not", function (done) {
+      React.render((
+        <Router history={createHistory('/website')} routes={routes} />
+      ), node, function () {
+        expect(node.querySelector('#overviewLink').className).toEqual('active')
+        expect(node.querySelector('#contactLink').className).toEqual('')
+        expect(node.querySelector('#productsLink').className).toEqual('')
+        expect(node.querySelector('#specificProductLink').className).toEqual('')
+        done()
+      })
+    })
+  })
+
+  describe('when linking to the contact', function () {
+    it("is active and other routes are not", function (done) {
+      React.render((
+        <Router history={createHistory('/website/contact')} routes={routes} />
+      ), node, function () {
+        expect(node.querySelector('#overviewLink').className).toEqual('')
```

Comment:
This expectation is not accurate. At `/website/contact` both the `Website` route *and* the `WebsiteContact` routes are active.